catalog/MDCCatalog/MDCCatalogTileView.swift

```diff
@@ -30,13 +30,12 @@ class MDCCatalogTileView: UIView {
       imageView.image = getImage(componentNameString)
     }
   }
-  let imageView = UIImageView()
-  let imageCache = NSCache<AnyObject, UIImage>()
+  private lazy var imageView = UIImageView()
```

Comment:
Same as above, why not set the frame inside the creation of the property?

Reply:
For this one, I don't want to set the frame in the property since it could happen at any moment. I'd rather have the frame set explicitly somewhere where I can predict the state of the view it depends on for that calculation.